src/js/components/Diagram/Diagram.js

```diff
@@ -36,13 +43,36 @@ const COMMANDS = {
         `Q ${fromPoint[0] + offset},${midPoint[1] + offset} ` +
         `${midPoint[0] + offset},${midPoint[1] + offset} `;
     }
-    cmds += `T ${toPoint[0] + offset},${toPoint[1] + offset}`;
+    const lineLength = ['top'].includes(elementPosition)
+      ? decreaseLineLength
+      : -decreaseLineLength;
```

Comment:
This code block is triplicated. 
Can you please optimize it to a constant length value and reuse it across the file?